requirements.txt

```diff
@@ -17,3 +17,4 @@ pyyaml
 redis
 sqlalchemy
 oslo.config==1.15.0
++-e git://github.com/mirrorcoder/paramiko.git#egg=paramiko
```

Comment:
Please add comments describing why we need this (link to paramiko bug would probably do the job)